docs/integrations/web-servers/opentelemetry/varnish-opentelemetry.md

```diff
@@ -0,0 +1,172 @@
+---
+id: varnish-opentelemetry
+title: Varnish - OpenTelemetry Collector
+sidebar_label: Varnish - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Varnish.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/web-servers/varnish.png')} alt="Thumbnail icon" width="75"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for [Varnish](https://varnish-cache.org/intro/) provides dashboards that help you analyze logs generated by Varnish servers. This app allows you to identify traffic sources, improve application and website workflows, and understand how customers use your product.
+
+Varnish logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Varnish-OpenTelemetry/Varnish-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Varnish
+
+- **`webengine.cluster.name`** - User configured. Enter a name to identify this Varnish cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.system`**  - Has a fixed value of **varnish**.
+- **`sumo.datasource`** - Has a fixed value of **varnish**.
+
+## Prerequisites
+
+1.  Configure logging in Varnish. Varnish supports logging via local text log files. For details please visit this [page](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/). For the dashboards to work properly, please set the Varnish log format as explained [here](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/#step-3-customise-options-1):
```

Comment:
Indent the code block under step 1 so that step 2 displays properly as "2".  See [Numbered lists](https://help.sumologic.com/docs/contributing/style-guide/#numbered-lists) in the style guide.